src/content/reference/react-dom/flushSync.md

```diff
@@ -4,13 +4,13 @@ title: flushSync
 
 <Pitfall>
 
-Using `flushSync` is uncommon and can hurt the performance of your app.
+Penggunaan `flushSync` tidak umum dan dapat menyebabkan penurunan kinerja aplikasi Anda.
 
 </Pitfall>
 
 <Intro>
 
-`flushSync` lets you force React to flush any updates inside the provided callback synchronously. This ensures that the DOM is updated immediately.
+`flushSync` dapat memaksa React untuk menge-*flush* pembaruan dalam *callback* yang diberikan secara sinkron.
```

Comment:
@mhaidarhanif @zainfathoni @mazipan Do we have a better translation for "flush" in the glossary? Or should we keep it like this for now?

Reply:
Possible translations:
- gelontor
- bilas
- dirus
- siram
- sentor

Reply:
I think we can keep it as "flush" for now. @aris-uu